Wiki-Style-Guide.md

```diff
@@ -1,19 +1,19 @@
-# What Does "Wiki" Means?
+# What Does “Wiki” Mean?
 
-"Wiki" is a Hawaiian word meaning "quick". Keep this in mind when creating wiki articles. We are always happy to help new contributors write, edit, and localize our wiki articles. Help us make Free Code Camp the best it can be.
+“Wiki” is a Hawaiian word meaning “quick.” Keep this in mind when creating wiki articles. We are always happy to help new contributors write, edit, and localize our wiki articles. Help us make Free Code Camp the best it can be!
 
-You can create a new wiki page by clicking the "New Page" button at top right.
+You can create a new wiki page by clicking the “New Page” button at top right.
 
 Keep your wiki articles short and focused.
 
 You can include images if they help you better communicate your subject.
 
-## Use descriptive, punctuation-free titles that follow a name convention as `Name of Page`.
+## Use descriptive, punctuation-free titles that follow the naming convention `Name of Page`.
 
-Our wiki articles are [written in GitHub-flavor Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+Our wiki articles are written in [GitHub-flavored Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
 
 - [Algorithm Style Guide](Algorithm-Style-Guide)
 
-If you have questions about contributing to the Free Code Camp Wiki message [@rafase282](https://gitter.im/rafase282) in [Gitter](https://github.com/FreeCodeCamp/freecodecamp/wiki/Gitter)
+If you have questions about contributing to the Free Code Camp Wiki, message [@rafase282](https://gitter.im/rafase282) in [Gitter](https://github.com/FreeCodeCamp/freecodecamp/wiki/Gitter).
```

Comment:
Although not concerned with this PR, 

```diff
- If you have questions about contributing to the Free Code Camp Wiki, message [@rafase282](https://gitter.im/rafase282) in [Gitter](https://github.com/FreeCodeCamp/freecodecamp/wiki/Gitter)
+ If you have any questions regarding contributing to the Free Code Camp Wiki, feel free to contact Wiki Moderators in  <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"><img src="http://i.imgur.com/ThSWa6Y.png?2"> <b>FreeCodeCamp/Wiki</b></a></kbd>
```
Which will appear as ...

If you have any questions regarding contributing to the Free Code Camp Wiki, feel free to contact Wiki Moderators in  <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"><img src="http://i.imgur.com/ThSWa6Y.png?2"> <b>FreeCodeCamp/Wiki</b></a></kbd>